qa/vagrant/src/main/java/org/elasticsearch/packaging/test/ArchiveTestCase.java

```diff
@@ -192,7 +194,70 @@ public void test52BundledJdkRemoved() throws IOException {
         }
     }
 
-    public void test60AutoCreateKeystore() {
+    public void test53JavaHomeWithSpecialCharacters() throws Exception {
+        assumeThat(installation, is(notNullValue()));
+
+        Platforms.onWindows(() -> {
+            final Shell sh = new Shell();
+            final String originalPath = sh.run("$Env:PATH").stdout.trim();
+            final String javaHome = sh.run("$Env:JAVA_HOME").stdout.trim();
```

Comment:
JAVA_HOME does not always exist. You need to look for SYSTEM_JAVA_HOME

Reply:
will remove that at all. As per your comment [here](https://github.com/elastic/elasticsearch/pull/39712/files#r270126523) we don't need to clean it up